public/css/index.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  background: #1e1e25;
+}
+
+.form {
+  position: absolute;
+  left: 0;
+  right: 0;
+  background: #27293d;
+  height: 90%;
+  width: 100%;
+  margin: 20px auto;
+  padding: 15px;
+  border-radius: 1%;
+}
+
+.form__label {
+  color: white;
+}
+
+.form__input {
+  background: none;
+  border: #2a324e solid 1px;
+  border-radius: 15px;
+  padding: 13px;
+  color: white;
+  width: 80%;
+}
+
+.form__input:focus {
+  border-color: #fb00ff;
+  background: #1d253b;
+  outline: none;
+}
+
+.form__field {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 40px;
+  position: relative;
+}
+
+.btn {
+  text-align: center;
+  padding: 10px 50px;
```

Comment:
paddingi i marginy (nie mogę teraz znaleźć artyk.) najlepiej jeśli są wielokrotnością 8px  4, 8, 16, 24, 32, 40, 48 itd

Reply:
@czechue hmm ciekawe. Jak byś kiedyś znalazł artykuł to podrzuć. Możliwe, że pomaga to jakoś na renderowanie przy różnych rozdzielczościach, kiedy nagle 1px potrafi zniknąć itp. Ale chętnie bym poczytał jakie jeszcze mogą być powody. Chyba, że pamiętasz co tam było :)

Reply:
powody są prozaiczne.
stosując paddingi i marginy np raz 7px raz 17px raz 22 raz 23 layout jest chaotyczny. a ludzkie oko lubi symetrię, linearność itd. to są tylko względy estetyczne ale grid odgrywa bardzo dużą rolę w odbiorze wizualnym.

to jeszcze nie jest dokładnie ten art o który mi chodziło, ale wnioski są bardzo podobne:
https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62